### PR TITLE
IGVF-401-add-dbxrefs-file-set-donor

### PR DESCRIPTION
**Pull request by jenjou**

### CODE REVIEW

File: src/igvfd/schemas/changelogs/measurement_set.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 ## Changelog for *`measurement_set.json`*
 
+### Minor changes since schema version 3
+
+* Add `dbxrefs`.
+
+
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/measurement_set.md:7]: extra line should be removed

File: src/igvfd/schemas/changelogs/file_set.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`file_set.json`*
 
+### Minor changes since schema version 2
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/file_set.md:3]: I thought we are converting changelogs to be date based? or is the assumption that the other ticket will cover this change? IGVF-417?

File: src/igvfd/schemas/file_set.json [hunk 3]
```diff
@@ -48,6 +48,21 @@
         "accession": {
             "accessionType": "DS"
         },
+        "dbxrefs": {
+            "@type": "@id",
+            "rdfs:subPropertyOf": "rdfs:seeAlso",
+            "title": "External Resources",
+            "description": "Identifiers from external resources that may have 1-to-1 or 1-to-many relationships with IGVF file sets.",
+            "comment": "This property is overwritten by the subclasses to define specific enum values.",
+            "type": "array",
```
Comment (gabdank) [inline src/igvfd/schemas/file_set.json:57]: minItems = 1

File: src/igvfd/schemas/donor.json [hunk 4]
```diff
@@ -41,6 +41,21 @@
         "accession": {
             "accessionType": "DO"
         },
+        "dbxrefs": {
```
Comment (gabdank) [inline src/igvfd/schemas/donor.json:44]: minItems = 1

**Review by gabdank — CHANGES_REQUESTED** [review]

File: src/igvfd/schemas/changelogs/file_set.md (hunk 2 above)
Comment (jenjou) [reply src/igvfd/schemas/changelogs/file_set.md:3]: I was not sure which ticket would proceed first, added this line assuming I would have to redo all the changes in rebasing.

**Review by gabdank — APPROVED** [review]

**Pull request merged**